app/Actions/Albums/Extensions/PublicIds.php

```diff
@@ -64,7 +64,7 @@ private function init(): Builder
 	 * Return a collection of Album that are not directly accessible by visibility criteria
 	 * ! we do not include password protected albums from other users.
 	 *
-	 * @return BaseCollection[(_lft, _rgt)]
+	 * @return BaseCollection<(_lft, _rgt)>
```

Comment:
I know, this is nitpicking. given the fact that this method will be gone soon. But the correct PHPdoc syntax for associative arrays uses curly braces and an UML-like style for properties. I.e. in this case: `BaseCollection<array{_lft: int, _rgt: int}>`

Reply:
```suggestion
	 * @return BaseCollection<array{_lft: int, _rgt: int}>
```